PyTorchEx1.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    bert = AutoModel.from_pretrained('bert-base-uncased')
</pre><p> </p><p>Replace with this model : </p><p><code></code><code></p><p>from transformers import AutoTokenizer, AutoModelForSequenceClassification</p><p>&nbsp;</p><p>model = AutoModelForSequenceClassification.from_pretrained("nlptown/bert-base-multilingual-uncased-sentiment")</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>